### PR TITLE
Impressions and calls

### PR DESCRIPTION
**Pull request by PriyankaSahani**
Added slides for impressions, ALP visits and calls page.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -42,6 +42,56 @@ <h1 style="color: white; font: 500 30px/36.6px Google Sans"> Your Google Ads jou
...
+                <span class="business-name">MeriBindiya</span>
+                <div class="icons group1"><img src="image/call.png" style="max-height: 100%;"></div>
+                <div class="icons group2"><img src="image/gmb.png" style="max-height: 100%;"></div>
+                <div class="icons group3"><img src="image/list.png" style="max-height: 100%;"></div>
+                <div class="content count call-count">200</div>
+                <div class="content text call-text">calls</div>
+                <div class="content count visit-count">150</div>
+                <div class="content text visit-text">store visits</div>
+                <div class="content count lead-count">100</div>
+                <div class="content text lead-text">submitted leads</div>
```
Comment (SaachiGrover) [inline index.html:122]: What are submitted leads in AdsGo context?

File: index.html [hunk 2]
```diff
@@ -42,6 +42,56 @@ <h1 style="color: white; font: 500 30px/36.6px Google Sans"> Your Google Ads jou
...
+        </div>
+
+        <div class="slide">
+            <div class="impressions" style="background-color: #244d52; position: relative; ">
+                <img src="https://fonts.gstatic.com/s/i/productlogos/google_my_business/v6/192px.svg" class="logo">
+                <span class="business-name">MeriBindiya</span>
+                <div class="icons group1"><img src="image/call.png" style="max-height: 100%;"></div>
+                <div class="icons group2"><img src="image/gmb.png" style="max-height: 100%;"></div>
+                <div class="icons group3"><img src="image/list.png" style="max-height: 100%;"></div>
+                <div class="content count call-count">200</div>
```
Comment (SaachiGrover) [inline index.html:117]: Can you work with @manojkumarsure to obtain exact numbers for **Buttercup.Co** (Business chosen for demo) to ensure we have a sizeable numbers in past month to report all metrics?

File: index.html (hunk 1 above)
Comment (PriyankaSahani) [reply index.html:122]: form fills

File: index.html (hunk 2 above)
Comment (PriyankaSahani) [reply index.html:117]: yup sure, we'll get a sample.

File: css/style.css [hunk 3]
```diff
@@ -525,3 +525,247 @@ body {
...
+}
+
+.content {
+  position: absolute;
+  left: 190px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  font-style: normal;
+  font-family: 'Open Sans', sans-serif;
```
Comment (SaachiGrover) [inline css/style.css]: Can you change font to **Google Sans** for consistency?

File: index.html [hunk 4]
```diff
@@ -42,6 +42,56 @@ <h1 style="color: white; font: 500 30px/36.6px Google Sans"> Your Google Ads jou
             </div>
         </div>
 
+        <div class="slide" style="background-color: #f0e9e9;">
```
Comment (SaachiGrover) [inline index.html:76]: Nit: We have a single white frame with dark text in the middle of dark frames. Can we change this to a dark overlay calendar/clicks image/video (eg. https://www.pexels.com/video/person-using-a-cellphone-6974194/) with white text?

File: js/script.js [hunk 5]
```diff
@@ -151,6 +151,24 @@ function setPositionByIndex() {
   if (currentIndex === 0) {
     animateTitle();
   }
+
+  if (currentIndex === 2) {
```
Comment (SaachiGrover) [inline js/script.js:155]: Nit: Animations do not restart when slide is visited a second time.

File: css/style.css (hunk 3 above)
Comment (PriyankaSahani) [reply css/style.css]: done.

**Pull request merged**